Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+ruby File.read('.ruby-version').strip if ENV["RACK_ENV"] == "production" # strict ruby version only on heroku
```

Comment:
why not locally ?
... I'd rather not run a different version when developing ...